app/views/patients/_patient.html.erb

```diff
@@ -14,10 +14,14 @@
     <a class="anchor" name="overview"></a>
     <div class="panel panel-default">
       <div class="panel-heading">
-        Patient Overview
+        Patient Summary
       </div>
       <div class="panel-body">
-        <%= "Name: #{@patient_data.full_name}" %>
+        <b> Name: </b><%= "#{@patient_data.full_name}" %>
+        <br>
+        <b> Weight: </b><%= "#{@patient_data.weight}"%> <br>
```

Comment:
If you're not adding anything to the string, you can just do this:

```erb
    <%= @patient_data.weight %>
```

Maybe you did at one point and then took it out?